src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carCost = 40;
+  let allCost = days * carCost;
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 2nd item
```suggestion
  const allCost = days * carCost;
```